make/firmware-defs.mk

```diff
@@ -1,3 +1,19 @@
+ifneq ($(BOARD_INFO_DIR)x,x)
+include $(BOARD_INFO_DIR)/board-info.mk
+
+USB_VID_EXPAND := $(shell echo -n "$(USB_VID)" | sed -re "s/(.)/'\1',/g" -e 's/,$$//')
```

Comment:
Should call these USB_VENDOR and USB_PRODUCT instead IMO. VID and PID are the 16-bit integers.